docs/src/main/sphinx/docutils.conf

```diff
@@ -0,0 +1,2 @@
+[html writers]
+table_style = colwidths-grid
```

Comment:
We should be able to move this to `conf.py` with all the other configs. Maybe `html_table_style`.

Reply:
I tried that and it didn't work.